### PR TITLE
Added docstrings to TestCompleteAnOrder

### PR DESCRIPTION
**Pull request by nchemsak**
## Description
Added docstrings to TestCompleteAnOrder

## Related Issue
[#6](https://github.com/Ambiguous-Squids/bangazon-cli/issues/6)

## Motivation and Context
Better documentation for this test module.

## How Has This Been Tested?
$ python TestCompleteAnOrder.py

## Steps to Test
$ python TestCompleteAnOrder.py

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x\] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x\] All new and existing tests passed.

### CODE REVIEW

File: tests/TestCompleteAnOrder.py [hunk 1]
```diff
@@ -9,39 +12,76 @@
...
     '''
 
     @classmethod
-    def setUpClass(self):
-        self.customer = Customer("Nick", "Chemsak", "111 Street Rd", "suite 3", "Nashville", "TN", "37075", "123-123-0987", "test@test.com")
-        self.order = Order("Nick", "Basketball", True, 1)
-        self.product = Product("Basketball", 5.00)
-        self.payment_option = PaymentOption("Nick", "Chemsak", "1234567891234567", "2017-05-05", "123", "VISA")
-        self.order2 = Order("Nick", "", True, 1)
+    def setUpClass(cls):
```
Comment (asimonia) [inline tests/TestCompleteAnOrder.py:32]: well done, Nick

**Review by rtwhitfield84 — APPROVED** [review]
👍

File: tests/TestCompleteAnOrder.py [hunk 2]
```diff
@@ -9,39 +12,76 @@
...
+            "1234567891234567",
+            "2017-05-05",
+            "123",
+            "VISA"
+            )
+        cls.order2 = Order("Nick", "", True, 1)
 
 
     def test_there_is_a_customer(self):
+        '''test that a customer exists'''
```
Comment (asimonia) [inline tests/TestCompleteAnOrder.py:58]: i ❤️ docstring

**Review by asimonia — APPROVED** [review]

**Comment by asimonia** [comment]
🗡

**Pull request merged**